build.bat

```diff
@@ -0,0 +1,15 @@
+REM Usage .\build.bat <Build Configuration> <VCPKG_DIR> <MSBUILD_DIR> <CMAKE_DIR>
+REM Example: .\build.bat Release C:\repos\vcpkg "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin" "C:\Program Files\CMake\bin"
+set Conf=%1
+set VCPKG_DIR=%2
+set MSBUILD_DIR=%3
+set CMAKE_DIR=%4
+IF exist build ( echo Build directory exists && cd build) ELSE ( 

```

Comment:
Please try to improve the indentation, and split into multiple lines, if batch scripts allow this...